test/parallel/test-dgram-udp6-link-local-address.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const common = require('../common');
+if (!common.hasIPv6)
+  common.skip('no IPv6 support');
+
+const assert = require('assert');
+const dgram = require('dgram');
+
+const os = require('os');
+const ifaces = os.networkInterfaces();
+
+// Try to an interface with an IPv6 link local address
+const address = Object.keys(ifaces)
+
+// Convert the interface names to the interface arrays of addresses
+      .map(
+        (ifname) => ifaces[ifname]
+          .map((ifobject) => Object.assign({ ifname: ifname }, ifobject)))
+
+  // Flatten the array of arrays
+      .reduce((prev, curr) => prev.concat(curr))
+
+  // Filter out all non-IPv6 addresses
+      .filter((iface) => iface.family === 'IPv6')
+
+  // Map the interface into its address, with interface ID
+      .map((iface) => iface.address + '%' + iface.ifname)
+
+  // Return the first one with a link-local address
```

Comment:
Can you line up the comments?  Their indentation is a bit inconsistent now.

Reply:
The lineup of the comments is what the JavaScript linter wants.  I don't like it myself; I just wanted to make the linter quiet.